src/nodes/Heading/HeadingView.vue

```diff
@@ -0,0 +1,112 @@
+<!--
+  - @copyright Copyright (c) 2022
+  -
+  - @license AGPL-3.0-or-later
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<NodeViewWrapper ref="content"
+		:as="domElement"
+		v-bind="node.attrs"
+		tabindex="-1">
+		<a aria-hidden="true"
+			class="anchor-link"
+			:href="href"
+			:title="t('text', 'Permalink')"
```

Comment:
I'm not sure Permalink is a good term here. For me Permalink means this link will stay available over time. However if the section or the file are renamed this link might change - obviously depending on the url used initially.

First ideas that come to my mind are 'Link to this section', 'Link here' or 'Jump here'.

Reply:
I reword it to make it more meaningful.